parquet-hadoop/src/main/java/org/apache/parquet/hadoop/example/GroupReadSupport.java

```diff
@@ -35,8 +37,21 @@ public org.apache.parquet.hadoop.api.ReadSupport.ReadContext init(
       Configuration configuration, Map<String, String> keyValueMetaData,
       MessageType fileSchema) {
     String partialSchemaString = configuration.get(ReadSupport.PARQUET_READ_SCHEMA);
-    MessageType requestedProjection = getSchemaForRead(fileSchema, partialSchemaString);
-    return new ReadContext(requestedProjection);
+    String removeColumns = configuration.get(DataMaskingUtil.DATA_MASKING_COLUMNS);
```

Comment:
It seems that only the example `ReadSupport` has enabled skipping masked columns. Does it mean that other ReadSupport implementations are required to apply the same approach? If yes, does the `AvroReadSupport` class require the change, too?